lib/ReactViews/Map/HelpButton/HelpButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import ViewState from "../../../ReactViewModels/ViewState";
+import Icon from "../../../Styled/Icon";
+import Text from "../../../Styled/Text";
+import Prompt from "../../Generic/Prompt";
+
+import Styles from "./help-button.scss";
```

Comment:
I think we should use styled components here instead of SCSS.

Reply:
i'm using help-button.scss in order to extend _buttons.scss and retain the existing styling of the existing buttons. I agree we should use styled components but i felt that de-scssing all the buttons in the app was a bit more work than i was willing to take on :/